jenkins/it/Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+pipeline {
+    agent any
+
+    triggers {
+        cron("H H(0-3) * * 2-6")
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "2"))
+        timeout time: 60, unit: "MINUTES"
+    }
+
+    stages {
+        stage("Build") {
```

Comment:
Does a stages setup reflect the current configuration of a `iis_integration_tests` Jenkins job:

http://ci.ceon.pl/view/IIS/job/iis_integration_tests

?

I am just wondering, again, whether we do need a separate `Build` stage here? It will introduce an additional overhead since in `Verify` stage the project will also be compiled.

Reply:
I've removed separate build stage, so now this pipeline contains only verification stage. This stage executes `install` phase with the same maven parameters as `iis_integration_tests` Jenkins job.

Jenkins job built using this Jenkinsfile will have the same settings as `iis_integration_tests`:
* log rotation with 2 previous builds
* periodic builds
* same email recipients

`iis_integration_tests` is parameterized by branch name and the replacement job will also be parameterized by branch name. However, this is done in Jenkins UI within job configuration settings, not in Jenkinsfile.